pkg/disttask/framework/dispatcher/dispatcher.go

```diff
@@ -492,7 +488,7 @@ func (d *BaseDispatcher) dispatchSubTask4Revert(meta []byte) error {
 		subTasks = make([]*proto.Subtask, 0, len(instanceIDs))
 		for _, id := range instanceIDs {
 			// reverting subtasks belong to the same step as current active step.
-			subTasks = append(subTasks, proto.NewSubtask(d.Task.Step, d.Task.ID, d.Task.Type, id, meta))
+			subTasks = append(subTasks, proto.NewSubtask(d.Task.Step, d.Task.ID, d.Task.Type, id, []byte("{}")))
```

Comment:
Will business need subtask meta to rollback?
ok for now.

Reply:
might be, but we don't have real rollback now, and current rollback design that generate a task on each node looks bad to me.

maybe we can delete it completely later

Reply:
LGTM

Reply:
> might be, but we don't have real rollback now, and current rollback design that generate a task on each node looks bad to me, rollback shouldn't depends on node
> 
> maybe we can delete it completely later

Maybe add some cleanup logic for each nodes' scheduler is better.

Reply:
what to clean? disk occupied by subtask? this should be done before subtask success/fail